maxim/logger/livekit/agent_session.py

```
@@ -31,19 +31,28 @@ def intercept_session_start(self: AgentSession, *args, **kwargs):
     )
     # getting the room_id
     room = kwargs.get("room", None)
```

Ruff (0.11.9)

Use `kwargs.get("room")` instead of `kwargs.get("room", None)`

Replace `kwargs.get("room", None)` with `kwargs.get("room")`

(SIM910)